### PR TITLE
Fix squiggle with editor and fsi.

### PR DESCRIPTION
**Pull request by KevinRansom**
Visual Studio 16.4 has a regression:

Fixes: #7950 

image

Fixed:
image

### CODE REVIEW

**Comment by dsyme** [comment]
Can we have a test for this?  (I don't understand why this wasn't caught by our existing language service testing, see bug)

**Comment by cartermp** [comment]
We most likely don't have a test for this then.

**Comment by KevinRansom** [comment]
@cartermp please feel free to add one.

**Comment by KevinRansom** [comment]
@cartermp  it was surprisingly hard to get a testcase that actually fails without the fix.  However, this one does.

File: fcs/FSharp.Compiler.Service.Tests/FSharp.Compiler.Service.Tests.fsproj [hunk 1]
```diff
@@ -101,5 +101,6 @@
     <Reference Include="UIAutomationTypes" />
     <ProjectReference Include="CSharp_Analysis\CSharp_Analysis.csproj" />
     <ProjectReference Include="..\FSharp.Compiler.Service.ProjectCracker\FSharp.Compiler.Service.ProjectCracker.fsproj" />
+    <ProjectReference Include="$(FSharpSourcesRoot)\fsharp\FSharp.Compiler.Interactive.Settings\FSharp.Compiler.Interactive.Settings.fsproj" />
```
Comment (dsyme) [inline fcs/FSharp.Compiler.Service.Tests/FSharp.Compiler.Service.Tests.fsproj]: @KevinRansom Do you remember why this line was needed? I assume the FCS testing failed if it wasn't present?

(The line causes the FCS build to also build the latest FSharp.Core, which isn't really meant to happen as FCS is only meant to depend on a public lower-versioned FSharp.Core. This is causing some issues in integrations for some of my PRs)

Comment (KevinRansom) [reply fcs/FSharp.Compiler.Service.Tests/FSharp.Compiler.Service.Tests.fsproj]: @dsyme,  I believe it is in support of the added test to ensure the squiggle is found.
     vsintegration/tests/UnitTests/FsxCompletionProviderTests.fs

Comment (dsyme) [reply fcs/FSharp.Compiler.Service.Tests/FSharp.Compiler.Service.Tests.fsproj]: Hmmm that test isn't included in the FCS testing

Comment (dsyme) [reply fcs/FSharp.Compiler.Service.Tests/FSharp.Compiler.Service.Tests.fsproj]: I suspect it might not be needed, will check

**Comment by KevinRansom** [comment]
Yes probably not needed then.  If we don't test it, it shouldn't be necessary.

**Pull request merged**